src/controllers/complianceController.ts

```
@@ -0,0 +1,109 @@
+        passkeys: true,
+        salaryBatches: true,
+        salarySchedules: true,
+      },
+    });
+
+    if (!user) {
+      throw new AppError("User not found", 404);
+    }
+
+    // Omit sensitive backend secrets like encrypted keys and passcode hashes before export
+    const { passcodeHash, encryptedStellarSecret, keyEncryptionHint, totpSecretEncrypted, ...safeUser } = user;
```

ESLint

[error] Replace `·passcodeHash,·encryptedStellarSecret,·keyEncryptionHint,·totpSecretEncrypted,·...safeUser` with `⏎······passcodeHash,⏎······encryptedStellarSecret,⏎······keyEncryptionHint,⏎······totpSecretEncrypted,⏎······...safeUser⏎···`

(prettier/prettier)